### PR TITLE
VAGOV-6019: Finalize no proxy and fail fast flags.

### PR DESCRIPTION
**Pull request by jonpugh**
Now that the fail fast and no proxy flag will cause the `npm build` to exit, we can use it instead of our shell grep hack.

### CODE REVIEW

**Comment by jonpugh** [comment]
UPDATE: Superseded by: #589

**Comment by ElijahLynn** [comment]
Previous failure was 
> Error: ENOENT: no such file or directory, scandir '/var/aegir/projects/cms/pr595/docroot/vendor/va-gov/vagov-content/pages'

http://devshop.cms.va.gov/node/22665

**Comment by ElijahLynn** [comment]
K, it is failing because it still wants that vagov-content repo we were using on the older PR.

**Comment by ElijahLynn** [comment]
There is the new detail we added, and interesting, it is a 200 response with ZERO HTTP Body. 

> Response code: 200
> Start of body: 
> End of body:

**Comment by ElijahLynn** [comment]
> There is the new detail we added, and interesting, it is a 200 response with ZERO HTTP Body.
> 
> > Response code: 200
> > Start of body:
> > End of body:

This is fixed now, it was a memory exhaustion and it was confusing because we didn't have php `error_log` set so there were no logs AND we were getting a 200. Next build won't have this issue.

**Comment by ElijahLynn** [comment]
Destroying and rebuilding anew.

File: config/sync/config_split.config_split.prod.yml [hunk 1]
```diff
@@ -9,6 +9,7 @@ folder: ../config/prod
 module:
   fast404: 0
   syslog: 0
+  simplesamlphp_auth: 0
```
Comment (ElijahLynn) [inline config/sync/config_split.config_split.prod.yml]: How did this get added?

**Comment by ElijahLynn** [comment]
Closing to destroy and rebuild anew.

**Comment by ElijahLynn** [comment]
I changed the base branch from `develop` to `master` as we switched our CD to be based on the GitHub Flow model (as opposed to Git Flow) and `develop` is now deprecated (https://github.com/department-of-veterans-affairs/va.gov-cms/pull/617). It may be best instead of fixing conflicts to rebase/replay your commits on top of current master:

`git fetch --all`
`git rebase origin/master`
`git push --force`

**Comment by jonpugh** [comment]
Commits merged to #589

**Pull request closed without merging**